### PR TITLE
Feature/add&remove

### PR DESCRIPTION
**Pull request by TheLuisAbarca**
## Feature/add&remove

In this branch, I've added the following changes:

1. Add a restructuration on code.
2. Add Feature of Drag&Drop.
3. Remove all hardcoded items from the tasks array.
4. Create a new JavaScript file for the new functionality.
5. Implement a function for adding a new task (add a new element to the array).
6. Implement a function for editing task descriptions.
7. Implement a function for the 'Clear all completed' button (use filter () method).
8. Set By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5). Add functionality that by Deleting a task should update all remaining items indexes, so they represent the current list order and are unique.
9. Add a function for deleting a task (remove an element from the array) currently not being used but for future uses.
10. Fix Html and CSS Linter errors.
11. Fix Javascript Linter errors.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/html/index.html [hunk 1]
```diff
@@ -21,14 +21,17 @@ <h5 class="col-10">Today's To Do</h5>
             </div>
             <div class="row pt-2 pb-2 border-top border-1 justify-content-center align-items-center">
                 <input id="addTask" type="text" name="task" class="col-10" placeholder="Add to your list...">
-                <button class="col-2 add-Task text-muted">
+                <button id="btnaddTask" class="col-2 add-Task text-muted">
                     <i class="fas fa-level-down-alt fa-rotate-90"></i>
                 </button>
             </div>
-            <ul id="task-list" class="">
+            <ul id="task-list" class="task-list">
```
Comment (mikenath223) [inline src/html/index.html:28]: - [ ] It is not clear how to delete a todo item kindly add a delete icon beside each todo item.

Comment (TheLuisAbarca) [reply src/html/index.html:28]: Hi @mikenath223, thank you for your feedback on my project. Right, now it's implemented on commit 1b78770.

**Review by EricMbouwe — APPROVED** [review]
### Hi @TheLuisAbarca, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**